src/Common/test-runtime/project.json

```diff
@@ -3,15 +3,18 @@
       "xunit.console.netcore": "1.0.3-prerelease-00921-01",
       "Microsoft.DotNet.BuildTools.TestSuite": "1.0.1-prerelease-01001-04",
       "Microsoft.xunit.netcore.extensions": "1.0.1-prerelease-01001-04",
-       "Microsoft.DotNet.xunit.performance.analysis": "1.0.0-alpha-build0040",
-       "Microsoft.DotNet.xunit.performance.analysis.cli": "1.0.0-alpha-build0040",
-       "Microsoft.DotNet.xunit.performance.runner.cli": "1.0.0-alpha-build0040",
-       "Microsoft.DotNet.xunit.performance.runner.Windows": "1.0.0-alpha-build0040"
+      "Microsoft.DotNet.xunit.performance.analysis": "1.0.0-alpha-build0040",
+      "Microsoft.DotNet.xunit.performance.analysis.cli": "1.0.0-alpha-build0040",
+      "Microsoft.DotNet.xunit.performance.runner.cli": "1.0.0-alpha-build0040",
+      "Microsoft.DotNet.xunit.performance.runner.Windows": "1.0.0-alpha-build0040",
+      "Microsoft.NETCore.Runtime.CoreCLR": "1.2.0-beta-24728-02",
+      "Microsoft.NETCore.TestHost": "1.2.0-beta-24728-02",
+      "Microsoft.NETCore.Windows.ApiSets": "1.0.2-beta-24727-00",
```

Comment:
I am pretty sure that the ApiSets are not required any longer by CoreClr, so can we remove them?